### PR TITLE
#118

### PR DESCRIPTION
**Pull request by Yurashk**

### CODE REVIEW

File: src/app/src/_domain/modules/profile/orders/orders.component.html [hunk 1]
```diff
@@ -15,7 +15,10 @@ <h3>Orders</h3>
             </div>
           </div>
         </div>
-        <div class="text-muted font-less">
+        <div class=" font-less"
+             [ngClass]="{'text-warning':item.status==='processing',
+                         'text-warning':item.status==='pending',
+                         'text-success':item.status==='completed '}">
```
Comment (ultramarine256) [inline src/app/src/_domain/modules/profile/orders/orders.component.html]: Use 'text-secondary' class for 'pending' status.

File: src/app/src/_presentation/pages/checkout/checkout-page.component.ts [hunk 2]
```diff
@@ -12,6 +12,7 @@ import {
   OrderResponse, UserRepository, UserModel
 } from '../../../_data';
 import {AuthService, CartItemEntity, CartService, ValidationHelper} from '../../../_domain';
+import {ProjectRepository, ProjectResponse} from '../../../_data/repository/project';
```
Comment (ultramarine256) [inline src/app/src/_presentation/pages/checkout/checkout-page.component.ts]: Simplify import path to "../../../_data"

File: src/app/src/_presentation/pages/checkout/checkout-page.component.ts [hunk 3]
```diff
@@ -31,6 +32,7 @@ export class CheckoutPageComponent implements OnInit {
   /// spinners
   public isLoading = false;
   public isUserLoaded = false;
+  public didLoaded = false;
```
Comment (ultramarine256) [inline src/app/src/_presentation/pages/checkout/checkout-page.component.ts]: Add new line before "/// helper"

File: src/app/src/_presentation/pages/checkout/checkout-page.component.ts [hunk 4]
```diff
@@ -40,19 +42,23 @@ export class CheckoutPageComponent implements OnInit {
...
-      new Project({id: 'pr-313-3', name: 'Project 313-3'}),
-    ];
+              private userRepository: UserRepository,
+              private projectRepository: ProjectRepository) {
+
+    // this.projects = [
+    //   new Project({id: 'pr-313-1', name: 'Project 313-1'}),
+    //   new Project({id: 'pr-313-2', name: 'Project 313-2'}),
+    //   new Project({id: 'pr-313-3', name: 'Project 313-3'}),
+    // ];
```
Comment (ultramarine256) [inline src/app/src/_presentation/pages/checkout/checkout-page.component.ts]: Remove comments if not needed

File: src/app/src/_presentation/pages/profile/projects/projects-page.component.ts [hunk 5]
```diff
@@ -17,7 +16,6 @@ export class ProjectsPageComponent {
     this.projectRepository.getOrders()
       .pipe(finalize(() => this.didLoaded = true))
       .subscribe((items: Array<ProjectResponse>) => {
-        console.log(items);
         this.projects = items;
       });
```
Comment (ultramarine256) [inline src/app/src/_presentation/pages/profile/projects/projects-page.component.ts]: Remove redundant Brackets:

.subscribe((items: Array<ProjectResponse>) => this.projects = items);

File: src/app/src/_domain/modules/profile/orders/orders.component.ts [hunk 6]
```diff
@@ -9,3 +9,5 @@ import {OrderResponse} from '../../../../_data';
 export class OrdersComponent {
   @Input() items: Array<OrderResponse> = [];
 }
+
+
```
Comment (ultramarine256) [inline src/app/src/_domain/modules/profile/orders/orders.component.ts]: Remove extra lines if not needed.

Comment (Yurashk) [reply src/app/src/_domain/modules/profile/orders/orders.component.ts]: Done.

File: src/app/src/_presentation/pages/checkout/checkout-page.component.ts (hunk 4 above)
Comment (Yurashk) [reply src/app/src/_presentation/pages/checkout/checkout-page.component.ts]: Done.

File: src/app/src/_presentation/pages/checkout/checkout-page.component.ts (hunk 3 above)
Comment (Yurashk) [reply src/app/src/_presentation/pages/checkout/checkout-page.component.ts]: Done.

**Pull request merged**